events/religion_events/holy_order_events.txt

```diff
@@ -377,7 +377,18 @@ holy_order.0202 = {
 	}
 
 	option = { #Pretty please give me more time?
-		name = holy_order.0202.b
+		name = {
+			trigger = {
+				like_christianity_religion_trigger = yes
+			}
+			text = holy_order.0202.b_christian
```

Comment:
` holy_order.0202.b_christian:0 "Is patience not a virtue?"`
I suggest using faiths with patient trait as virtue here instead

Reply:
Done in 542bb930e0cf78ca86a1bd14160750c8108796da.